ebuild-maintenance/general/text.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0"?>
+<guide self="ebuild-maintenance/general/">
+<chapter>
+<title>Maintaining Your Package</title>
+
+<section>
+<title>General responsibilities</title>
+<body>
+
+<note>
+Gentoo's Quality Assurance (QA) team also maintains a
+<uri link="https://projects.gentoo.org/qa/policy-guide/">policy guide</uri>,
+which contains useful advice on ebuild structure and behaviour.
+</note>
+  
+<p>
+You should always use the latest <uri link="::ebuild-writing/eapi/">EAPI</uri>.
+
```

Comment:
If we are going to make this canon, I'd like this to be more precise:
```suggestion
You should always use the latest <uri link="::ebuild-writing/eapi/">EAPI</uri>
when adding new ebuilds or eclasses to ::gentoo repository.
```
Because it's fine to not bump EAPI when you add a patch, or eg a subslot binder to a dep with new subslot. Of course this is also common sense.